lib/cli.rb

```diff
@@ -337,13 +337,14 @@ def set_priorities
 EOT
   option 'board-id', desc: 'Id of the board', required: true
   option 'target-board-id', desc: 'Id of the target board', required: true
+  option 'labelled', desc: 'Set true to label cards as - in the the last sprint', default: 'False'
```

Comment:
This should be of type `boolean`. Check [thor documentation](https://github.com/erikhuda/thor/wiki/Method-Options#types-for-method_options) for details.